src/test/test_utils.cpp

```diff
@@ -374,6 +382,42 @@ void prepareKFSInferInputTensor(::KFSRequest& request, const std::string& name,
         data, putBufferInInputTensorContent);
 }
 
+void prepareCAPIInferInputTensor(ovms::InferenceRequest& request, const std::string& name, const std::tuple<ovms::shape_t, const ovms::Precision>& inputInfo,
+    const std::vector<float>& data, bool putBufferInInputTensorContent, bool badSize) {
+    auto [shape, type] = inputInfo;
+    prepareCAPIInferInputTensor(request, name,
+        {shape, ovmsPrecisionToPrecision(type)},
+        data, putBufferInInputTensorContent, badSize);
+}
+
+void prepareCAPIInferInputTensor(ovms::InferenceRequest& request, const std::string& name, const std::tuple<ovms::shape_t, const OVMSDataType>& inputInfo,
+    const std::vector<float>& data, bool putBufferInInputTensorContent, bool badSize) {
+    auto [shape, datatype] = inputInfo;
+    size_t elementsCount = 1;
+    for (auto const& dim : shape) {
+        ASSERT_GE(dim, 0);
+        elementsCount *= dim;
+    }
+
+    request.addInput(name.c_str(), datatype, shape.data(), shape.size());
+
+    if (!putBufferInInputTensorContent) {
```

Comment:
In TFS/KFS this boolean was needed to decide where to put data - tensor content or val containers. Our C-API does not have multiple places for content storage therefore I think we should get rid of this boolean.

Reply:
Removed.